README.md

```diff
@@ -159,6 +159,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [Adobe Photoshop CC](http://www.adobe.com/products/photoshop.html)
 - [Adobe Photoshop Lightroom CC](https://www.adobe.com/products/photoshop-lightroom.html)
 - [Airmail](http://airmailapp.com/)
+- [Alfred 3](https://www.alfredapp.com/)
```

Comment:
Alfred has been removed because they have a paid feature to support syncing. I don't think the devs would like Mackup to take this feature over. But feel free to ask them.